app/controllers/admin/notes_controller.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module Admin
+  class NotesController < ApplicationController
+    def create
+      @submission = Submission.find(note_params[:submission_id])
+      note = Note.create(gravity_user_id: @current_user, **note_params)
+      unless note.persisted?
+        flash[:notice] =
+          "Could not create note: #{note.errors.full_messages.join(', ')}"
+      end
+
+      redirect_to admin_submission_path(@submission)
```

Comment:
I think it is more informative to use the `notice` flash when the request is successful and use the `error` flash when it fails to create one.

```ruby
if note.save
  redirect_to admin_submission_path(@submission), notice: "Note has successfully been created."
else
  redirect_to admin_submission_path(@submission), error: "Could not create note: #{note.errors.full_messages.join(', ')}"
end
```

Reply:
there doesn't seem to be an `error` flash on the layout though.

Reply:
seems like `alert` was the right one:

```ruby
ApplicationController._flash_types
# => [:alert, :notice]
```